app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb

```diff
@@ -19,7 +20,14 @@ def raw_stop
 
   def raw_reboot_guest
     with_provider_connection(:service => 'PowerIaas') do |power_iaas|
-      power_iaas.reboot_pvm_instance(ems_ref)
+      power_iaas.soft_reboot_pvm_instance(ems_ref)
+    end
+    update!(:raw_power_state => "off")
+  end
+
+  def raw_reset
+    with_provider_connection(:service => 'PowerIaas') do |power_iaas|
+      power_iaas.hard_reboot_pvm_instance(ems_ref)
     end
     update!(:raw_power_state => "off")
```

Comment:
Is "off" the correct power state for `reset`?  If anything I think it'd be something like restarting or just still "ACTIVE"

Reply:
You're right @agrare. I just fired up my `while true;` looper and the state remains `ACTIVE` during resets.